src/KafkaFlow.Http/JwtAuthenticationHandler.cs

```diff
@@ -0,0 +1,89 @@
+﻿using KafkaFlow.Http.Options;
+
+using Microsoft.Extensions.Options;
+
+using System.Net.Http.Headers;
+using System.Text.Json;
+
+namespace KafkaFlow.Http;
+
+internal class JwtAuthenticationHandler<TKey, TValue> : DelegatingHandler
```

Comment:
@eduardosbcabral , na lifecycle-api, por exemplo, a autenticação é usando `client_secret` e `client_id`?

Reply:
Sim, atualmente nos autenticamos dessa forma. Podemos depois implementar algo mais genérico ou outros tipos de autenticação dependendo do que for necessário e extender o uso por parte do http client.